server/src/services/initialization/indexWorkspaceFolders.ts

```diff
@@ -22,10 +22,6 @@ export async function indexWorkspaceFolders(
   const logger = _.clone(serverState.logger);
   logger.tag = "indexing";
 
-  if (workspaceFolders.some((wf) => wf.uri.includes("\\"))) {
```

Comment:
Happy to see the check gone. Is it the case now that the workspace folders uri are not converted over to unix style?

Reply:
As part of this PR there were some places where the `convertToUnixStyle` was removed. Eventually I'd like to get rid of that across the codebase and use uris/paths being agnostic of their representation. I think the helper method is no longer needed if we use the `vscode-uri` package

Reply:
I am happy with that. I think the main push in that direction originally was trying to get consistent results for the test suite.

If we are happy we are covered by the protocol test suite around uri then this is a good move.